docs/test-config.md

```diff
@@ -52,6 +52,8 @@ configurations:
       run_time: str # (1)!
       # * Test target URL 
       target_host: str
+      # * Test stop timeout
+      stop_timeout: str
```

Comment:
One additional point: there is no indication to the unit here. I know it is in seconds, but the user may not know that.

Reply:
fixed